packages/gatsby-plugin-utils/src/polyfill-remote-file/__tests__/gatsby-image-resolver.ts

```diff
@@ -726,4 +729,30 @@ describe(`gatsbyImageData`, () => {
 
     expect(constrainedResult?.images.sources.length).toBe(2)
   })
+
+  it(`should call add image source urls to the redux store`, async () => {
+    fetchRemoteFile.mockResolvedValueOnce(
+      path.join(__dirname, `__fixtures__`, `dog-portrait.jpg`)
+    )
+    const actions = {
+      addGatsbyImageSourceUrl: jest.fn(),
+    } as Actions
+
+    process.env.GATSBY_SHOULD_TRACK_IMAGE_CDN_URLS = `1`
+
+    await gatsbyImageResolver(
+      portraitSource,
+      {
+        layout: `fixed`,
+        width: 300,
+      },
+      actions
+    )
+
+    process.env.GATSBY_SHOULD_TRACK_IMAGE_CDN_URLS = null
+
+    expect(actions.addGatsbyImageSourceUrl.mock.calls[0][0]).toEqual(
```

Comment:
Can we use `.toHaveBeCalledWith()` instead of accessing the calls directly?

Reply:
totally - will fix this up!